### DISCUSSION TITLE
v2ray 的tls指紋JA3S可能被識別了？

### DISCUSSION THREAD

**Discussion by nursery01** [v2fly/v2ray-core#2369] [Ideas]
加上cdn之後tls指紋特徵就是cdn的tls指紋特徵了，所以沒事

**Comment by 1265578519** [comment]
加上cdn后，墙已经不知道你的服务器真实ip了。正如v2开发者表示 被墙的都是没用cdn隐藏ip保护

**Reply by xiaosteven** [reply]
但是cdn最近高峰时段越来越不稳定，cloudflare用的人太多了，也不知道有没有其他的可用的cdn

**Reply by nursery01** [reply]
用收費的akamai，它可能是全球最大的cdn公司

**Reply by xiaosteven** [reply]
我看了下这个， 好像需要联系他们的销售才能购买使用，不能自己申请...请问下这个价格大概多少？

**Reply by nursery01** [reply]
我這邊是微軟azure的控制面板裡看見的，Akamai歐美靜態資料1GB是0.08美元，動態資料1GB是0.18美元。Akamai會不會直接收小顧客我不清楚，不過可以通過azure買Akamai的服務。其實我自己用的也是免費的cloudflare😅，Akamai手頭的顧客都是世界企業top500

**Reply by xiaosteven** [reply]
怪不得，我看了下他们只提供企业级的服务，看来还是要继续找稳定的cdn方案了。。。

**Reply by MakeFortuneQuietly** [reply]
免费的Gcore据说不行了，被墙直接封中转的ip
https://github.com/net4people/bbs/issues/216

**Reply by xiaosteven** [reply]
Gcore，几个月前在深圳电信这边好像就不行了，前几周是整体封掉了。 亚马逊的cdn也不太性，cf只能说勉强使用。

**Reply by MakeFortuneQuietly** [reply]
最后可能就只剩国内实名cdn可用。好比悬在头上的剑，让你知道它随时可以收网，所以就算用你也不敢搞事
